plugins/crd/validator/l3/l3_validator.go

```diff
@@ -124,13 +156,17 @@ func (v *Validator) validateVrf1PodRoutes(node *telemetrymodel.Node, vrfMap map[
 			continue
 		}
 
+		// Assume that the route will be valid. Each failed check flips
+		// the status
+		routeMap[1][lookUpRoute.Ipr.DstAddr] = routeValid
```

Comment:
I think it would be better to use var or const instead of magic number.

Reply:
routeValid is a constant - or did you want something else?